sentry-ruby/spec/sentry/net/http_spec.rb

```diff
@@ -9,6 +9,21 @@
     ::Logger.new(string_io)
   end
 
+  context "with IPv6 addresses" do
+    before do
+      perform_basic_setup
+    end
+
+    it "correctly parses the short-hand IPv6 addresses" do
+      stub_normal_response
+
+      conn = Net::HTTP.new('::1', 8080)
+      expect do
+        conn.request_post('/path', 'foo=1')
+      end.not_to raise_error
+    end
```

Comment:
It's a bit nitpicking, but I still prefer to test things as much end-to-end as possible:

```suggestion
    it "correctly parses the short-hand IPv6 addresses" do
      stub_normal_response

      transaction = Sentry.start_transaction
      Sentry.get_current_scope.set_span(transaction)

      _ = Net::HTTP.get("::1", "/path", 8080)

      expect(transaction.span_recorder.spans.count).to eq(2)

      request_span = transaction.span_recorder.spans.last
      expect(request_span.data).to eq(
        { "url" => "http://[::1]/path", "http.request.method" => "GET", "http.response.status_code" => 200 }
      )
    end
```

Reply:
Not at all — I think you're right, and I'm learning from you _a lot_ with each and every one of your comments and suggestions. Thank you for helping me learn!

Reply:
Thank you! I learn from your PRs too (such as this one) 😄